docs/source/guides/.pages

```diff
@@ -4,4 +4,5 @@ nav:
   - introduction-to-event-based-programming.md
   - development-workflows.md
   - creating-a-mod.md
+  - testing.md
```

Comment:
This isn't necessary: the `...` below means add all the other files from this folder to the navigation after the ones explicitly listed. For more info see [here](https://github.com/MWSE/MWSE/tree/master/docs#customizing-navigation).